### PR TITLE
room-sharing page complete & also in the dropdown

### LINKED ISSUE #61: Create ride/room share page
The new site design didn't include a page for the ride and room sharing page that we've had in the past. It's probably a good idea to add it.
- https://2023.djangocon.us/room-sharing/
- https://github.com/djangocon/2023.djangocon.us/blob/main/_pages/room-sharing.md

### PR DESCRIPTION
**Pull request by blulady**
### addresses issue #61
- modified venue drop down menu to include hrefs for other parts of the venue page like in 2023
- modified venue drop down menu to include href for the room/ride share page
- created room/ride share based on a combination of the visa & hackathon pages

### CODE REVIEW

File: src/room-sharing.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+      <p>Do you want to share a room at <a href="/venue/#hotel">one of the hotels</a> with other attendees, or do you have local accommodation available? Do you want to drive to DjangoCon US from Charlotte, Greensboro, Raleigh, or other relatively nearby cities?</p>
+
+      <p>We have a <a href="https://docs.google.com/spreadsheets/d/1487pyAR--sN3dpSDrDQWAQmxq25BgHIVM99xZIHRpp4/edit?usp=sharing">Google sheet</a> just for you! There are tabs for people who already have room reservations and are looking for roommates, who haven't yet booked anything, who live locally and want to offer their homes as accommodation, and who will be driving to Durham and need car buddies.</p>
+      <p>If you're looking for a room, list your contact information along with:</p>
+
+      <ul>
+      <li>How many other people you want to share with</li>
+      <li>What nights you'll be there</li>
+      <li>Wheather you've already reserved a room</li>
```
Comment (tim-schilling) [inline src/room-sharing.html]:
```suggestion
      <li>Whether you've already reserved a room</li>
```

File: src/room-sharing.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      <li>Any other requirements (e.g. man/woman, smoking/non, snoring/non, etc.)</li>
```
Comment (tim-schilling) [inline src/room-sharing.html]:
```suggestion
      <li>Any other requirements (e.g. gender, smoking/non, snoring/non, etc.)</li>
```

**Review by tim-schilling — COMMENTED** [review]
~~I have not yet tested this locally to confirm the page.~~ Everything's working!

File: src/room-sharing.html (hunk 1 above)
Comment (blulady) [reply src/room-sharing.html]: Thank you!!

File: src/room-sharing.html (hunk 2 above)
Comment (blulady) [reply src/room-sharing.html]: thank you!!

**Review by tim-schilling — APPROVED** [review]

**Comment by tim-schilling** [comment]
Thank you @blulady!

**Pull request merged**